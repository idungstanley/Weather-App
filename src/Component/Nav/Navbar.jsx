import React from 'react';
import { Link } from 'react-router-dom';
import './nav.css';
import logo from '../../Assets/weather-logo.png';

const Navbar = () => (
  <div className="container">
    <img src={logo} alt="logo" />
    <ul className="link-container">
      <li className="link">
        <Link to="/">Home</Link>
      </li>
    </ul>
  </div>

);

export default Navbar;
