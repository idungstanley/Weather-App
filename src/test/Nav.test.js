import React from "react";
import renderer from "react-test-renderer";
import { BrowserRouter as Router } from "react-router-dom";
import { Provider } from "react-redux";
import store from "../Redux/store";
import Navbar from "../Component/Nav/Navbar";

describe("Render components", () => {
  test("render Nav", () => {
    const tree = renderer
      .create(
        <Router>
          <Provider store={store}>
            <Navbar />
          </Provider>
        </Router>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
