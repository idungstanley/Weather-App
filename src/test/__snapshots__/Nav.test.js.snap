// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render components render Nav 1`] = `
<div
  className="container"
>
  <img
    alt="logo"
    src="weather-logo.png"
  />
  <ul
    className="link-container"
  >
    <li
      className="link"
    >
      <a
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
    </li>
  </ul>
</div>
`;
